climpred/comparisons.py

```diff
@@ -213,14 +176,12 @@ def _m2c(ds, control_member=None, metric=None):
         xr.object: forecast, reference.
     """
     if control_member is None:
-        control_member = [0]
-    reference = ds.isel(member=control_member).squeeze()
+        control_member = ds.member.values[0]
```

Comment:
Might want to make this `ds["member"][0]` so as not to load it into memory. I know it's a small thing to load into memory but might trip up dask. You could instead keep it lazy

Reply:
Coords are loaded anyways.